### PR TITLE
Fix problem with user/meeting_ids of merged users in meeting.import.

### PR DESCRIPTION
**Pull request by reiterl**
Use a less deep change to fix the problem. 
Add the test from the issue to show the problem.

### CODE REVIEW

**Review by r-peschke — CHANGES_REQUESTED** [review]

**Review by r-peschke — APPROVED** [review]

File: tests/system/action/meeting/test_import.py [hunk 1]
```diff
@@ -1341,6 +1341,56 @@ def test_merge_users_check_committee_and_meeting(self) -> None:
         meeting2 = self.assert_model_exists("meeting/2", {"committee_id": 1})
         assert sorted(meeting2.get("user_ids", [])) == [1, 14, 15, 16]
 
+    def test_merge_users_check_user_meeting_ids(self) -> None:
```
[the remark this answers is not in the document]
Comment (r-peschke) [reply tests/system/action/meeting/test_import.py:1413]: > Please, copy the whole test from #1369. It is still the test `test_merge_users_check_committee_and_meeting`, but with more logic and it tests also the committee_ids, that are wrong, too.
I attended the OS3/OS4 workings and fixed the tests. Finally spent more time and fixed the rest also. Now you could review:-)

File: openslides_backend/action/actions/meeting/import_.py [hunk 2]
```diff
@@ -644,6 +644,9 @@ def handle_calculated_fields(
                         process_calculated_fields_only=True,
                     )
                 )
+        for field, entry in relations.items():
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/meeting/import_.py:653]: Outdated.

File: tests/system/action/meeting/test_import.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting/test_import.py:1413]: Outdated.

**Review by r-peschke — APPROVED** [review]
Added a comment as hint for the new code part.
If you don't have more changes, you can merge, otherwise ask again for review

**Pull request merged**